From 52900a94a3231ff36f8e372477fe53fd68c4dbf8 Mon Sep 17 00:00:00 2001
From: Samuel Attard <sattard@slack-corp.com>
Date: Tue, 23 Apr 2019 09:51:43 -0700
Subject: [PATCH] build_add_electron_tracing_category.patch

All TRACE events in Chromium use a static assert to ensure that the
categories in use are known / declared.  This patch is required for us
to introduce a new Electron category for Electron-specific tracing.
---
 base/trace_event/builtin_categories.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/base/trace_event/builtin_categories.h b/base/trace_event/builtin_categories.h
index b2e301f218a48..edc1eb0edf2d2 100644
--- a/base/trace_event/builtin_categories.h
+++ b/base/trace_event/builtin_categories.h
@@ -80,6 +80,7 @@
   X("drmcursor")                                                         \
   X("dwrite")                                                            \
   X("DXVA_Decoding")                                                     \
+  X("electron")                                                          \
   X("evdev")                                                             \
   X("event")                                                             \
   X("event_latency")                                                     \
-- 
2.39.2

